aspnetcore/host-and-deploy/iis/protocols.md

```diff
@@ -10,19 +10,16 @@ no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blaz
 uid: host-and-deploy/iis/protocols
 ---
 
-# Using HTTP/2 with IIS
+# Using ASP.NET Core with HTTP/2 on IIS
```

Comment:
@guardrex  This might attract too much php and other non ASP.NET Core traffic otherwise.

Reply:
There shouldn't be a "Using" there. How about ...

```suggestion
# Use ASP.NET Core with HTTP/2 on IIS
```